__tests__/app.test.js

```diff
@@ -378,4 +352,27 @@ describe("GET /api/articles?topic", () => {
             })
         })
     })
+})
+
+describe("GET /api/articles/:article_id (comment_count)", () => {
+    test("status 200: responds with a specified article with added property of comment_count", () => {
+        return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((res) => {
+            expect(res.body).toMatchObject( {
+                article: {
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: expect.any(Number),
+                    body: expect.any(String),
+                    topic: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    article_img_url: expect.any(String),
+                    comment_count: expect.any(Number)
+                }
```

Comment:
A previous test will already be asserting a lot of this we only really care about the comment_count property being added here